ksqldb-rest-app/src/main/java/io/confluent/ksql/api/server/ServerUtils.java

```diff
@@ -97,6 +97,20 @@ public static String convertCommaSeparatedWilcardsToRegex(final String csv) {
     return out.toString();
   }
 
+  // See ParserUtil#getIdentifierText()
+  public static String getIdentifierText(final String text) {
+    if (text.isEmpty()) {
+      return "";
+    }
+
+    final char firstChar = text.charAt(0);
+    if (firstChar == '`' || firstChar == '"') {
```

Comment:
Do we really also support double quotes for escaping? Thought it was just backtick.